app/controllers/nonsipp/employercontributions/EmployerContributionsMemberListController.scala

```diff
@@ -216,47 +216,49 @@ object EmployerContributionsMemberListController {
             }
           ),
           TableElemDoubleLink(
-            // Change link
-            (mode, optYear, optCurrentVersion, optPreviousVersion) match {
-              case (ViewOnlyMode, Some(year), Some(currentVersion), Some(previousVersion)) =>
-                TableElem.view(
-                  controllers.nonsipp.employercontributions.routes.EmployerContributionsCYAController
-                    .onPageLoadViewOnly(
-                      srn,
-                      memberWithEmployerContributions.memberIndex,
-                      page = 1,
-                      year = year,
-                      current = currentVersion,
-                      previous = previousVersion
-                    ),
-                  Message(
-                    "employerContributions.MemberList.remove.hidden.text",
-                    memberWithEmployerContributions.employerFullName
+            (
```

Comment:
Just double-checking, this was only because of compilation warning, nothing logical was changed here, right?

Reply:
Yah because of compilation error, that needed to be addressed so I can run the tests and successfully compile